services-js/commissions-search/src/ResultList.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class ResultList extends Component {
```

Comment:
It would be nice to use React PropTypes (or TypeScript) to make it clear what these components are expecting.

Reply:
I'll let you guys handle this.